### PR TITLE
Add function to hide and show the matlab engine

### PR DESCRIPTION
**Pull request by musm**
For one this can be useful for debugging.

Also trim leading whitespaces.

### CODE REVIEW

**Comment by musm** [comment]
thoughts?

**Comment by musm** [comment]
The two added functions are named
`show_msession`
`hide_msession`

File: src/engine.jl [hunk 1]
```diff
@@ -59,7 +59,7 @@ function close(session::MSession)
     # Close a MATLAB Engine session
     @assert libeng::Ptr{Void} != C_NULL
     ret = ccall(engfunc(:engClose), Cint, (Ptr{Void},), session)
-    ret != 0 && throw(MEngineError("Failed to close a MATLAB engine session (err = $r)"))
+    ret != 0 && throw(MEngineError("failed to close a MATLAB engine session (err = $ret)"))
```
Comment (yuyichao) [inline src/engine.jl]: Note the extra space before `)`.

Comment (yuyichao) [inline src/engine.jl]: `engfunc(:engSetVisible )` below. Apparently @github puts the comment on the wrong line if it was made with whitespace changes removed. You can see it at the right line with https://github.com/JuliaInterop/MATLAB.jl/pull/79/files?w=0 .

Comment (musm) [inline src/engine.jl]: thanks again

**Comment by musm** [comment]
The only problematic issue here is if the user closes the matlab engine window. Because then the pointer to the engine is still  attached to a session that does not exist. I'm not sure what the best way to prevent such an occurrence.

**Pull request merged**